src/main/cpp/auton/drivePrimitives/DrivePathPlanner.cpp

```diff
@@ -97,6 +97,7 @@ void DrivePathPlanner::Init(PrimitiveParams *params)
     m_choreoTrajectoryName = params->GetTrajectoryName();
     m_pathGainsType = params->GetPathGainsType();
 
+    (DragonDataLoggerSignals::StringSignals::AUTON_PATH_NAME, m_pathname);
```

Comment:
Looks like you missing the method name

Should be LogStringData(DragonDataLoggerSignals::StringSignals::AUTON_PATH_NAME, m_pathname);